src/js/edu-benefits/5495/config/form.js

```diff
@@ -13,7 +13,7 @@ import fullNameUI from '../../../common/schemaform/definitions/fullName';
 
 import * as personId from '../../../common/schemaform/definitions/personId';
 
-import IntroductionPage from '../components/IntroductionPage';
+import IntroductionPage from '../components/SIPIntroductionPage';
```

Comment:
I think you can just update the current intro page. It's easy enough to just revert this commit if we need to.